src/gsi/rapidrefresh_cldsurf_mod.f90

```diff
@@ -418,6 +444,19 @@ subroutine init_rapidrefresh_cldsurf
     l_saturate_bkCloud= .true.
     l_rtma3d            = .false.                     ! turn configuration for rtma3d off          
     i_precip_vertical_check = 0                       ! No check and adjustment to retrieved Qr/Qs/Qg (default)
+    corp_howv           = 0.42_r_kind                 ! 0.42 meters (default)
+    hwllp_howv          = 170000.0_r_kind             ! 170,000.0 meters (170km as default for 3DRTMA, 50km is used in 2DRTMA)
+    i_howv_3dda         = 0                           ! no analysis of significant wave height (howv) in 3D analysis (default)
+
+!-- searching for speficif variable in state variable list (reading from anavinfo)
```

Comment:
specific

Reply:
@MatthewMorris-NOAA 
will correct this typo.
Thank you!